packages/@aws-cdk/aws-autoscaling/lib/auto-scaling-group.ts

```diff
@@ -1189,6 +1458,47 @@ export class AutoScalingGroup extends AutoScalingGroupBase implements
     return this.newInstancesProtectedFromScaleIn === true;
   }
 
+  /**
+   * Allows specify security group connections for instances of this fleet.
+   */
```

Comment:
Can we improve this comment to explain how this property it used, or remove it entirely?

Reply:
These getters were class properties. I copy-pasted their doc as is. To keep the backward compatibility I cannot really remove or rename them.

I can have a try improving the doc here in the next revision.

Reply:
Also this implements `IConnections`. It is a common practice to document something like _This property implements `IConnections`_?

Reply:
If the addition of the property is intended to implement `IConnections`, then you can add `implements IConnections` to the class declaration.

Reply:
It had been there before this change.

Reply:
Ah. No need to add a comment stating that the property implements `IConnections` then.